packages/app/components/Queue/TA/TAQueueDetailButtons.tsx

```diff
@@ -135,6 +161,15 @@ export default function TAQueueDetailButtons({
             </span>
           </Tooltip>
         </Popconfirm>
+        <Tooltip title="Ask the student to rework their question">
```

Comment:
Should this say `"Ask the student to add more detail to their question"` so it matches with what we show the student when the button is pressed?